packages/ag-charts-website/src/features/gallery/components/GalleryExampleRunner.tsx

```diff
@@ -90,37 +92,54 @@ const GalleryExampleRunnerInner = ({ title, exampleName, loadingIFrameId }: Prop
     }, [contents, exampleFilesIsLoading, exampleFilesIsError]);
 
     useEffect(() => {
-        setPlunkrHtmlUrl(
+        setHtmlUrl(
             getExampleWithRelativePathUrl({
                 exampleName,
             })
         );
     }, [exampleName]);
 
-    // Override `index.html` with generated file as
-    // exampleFiles endpoint only gets the index html fragment
     useEffect(() => {
         if (!contents || exampleFilesIsLoading || exampleFilesIsError || !exampleFileHtml) {
             return;
         }
         const files = {
             ...contents.files,
+            'package.json': contents.packageJson,
+            // Override `index.html` with generated file as
+            // exampleFiles endpoint only gets the index html fragment
             'index.html': exampleFileHtml,
         };
 
         setExampleFiles(files);
+        setPackageJson(contents.packageJson);
         setExampleBoilerPlateFiles(contents.boilerPlateFiles);
     }, [contents, exampleFilesIsLoading, exampleFilesIsError, exampleFileHtml]);
 
-    const externalLinkButton =
-        exampleFiles && plunkrHtmlUrl ? (
-            <OpenInPlunkr
-                title={title}
-                files={exampleFiles}
-                plunkrHtmlUrl={plunkrHtmlUrl}
-                boilerPlateFiles={exampleBoilerPlateFiles}
-                fileToOpen={initialSelectedFile!}
-            />
+    const externalLinks =
+        exampleFiles && htmlUrl ? (
+            <>
+                <li>
+                    <OpenInCodeSandbox
+                        title={title}
+                        files={exampleFiles}
+                        htmlUrl={htmlUrl}
+                        internalFramework={internalFramework}
+                        boilerPlateFiles={exampleBoilerPlateFiles}
+                        packageJson={packageJson!}
+                    />
+                </li>
+                <li>
+                    <OpenInPlunkr
+                        title={title}
+                        files={exampleFiles}
+                        htmlUrl={htmlUrl}
+                        boilerPlateFiles={exampleBoilerPlateFiles}
+                        packageJson={packageJson!}
```

Comment:
Does plunker need the package json?

Reply:
Yep, without it, plunkr doesn't know about where the TS types come from. So it enables autocomplete with it

Reply:
Oh nice